package.json

```diff
@@ -320,10 +319,10 @@
         "uniqid": "^5.2.0",
         "uuid": "^8.3.2",
         "wait-on": "^3.3.0",
-        "webpack": "^4.47.0",
-        "webpack-bundle-analyzer": "^3.6.0",
-        "webpack-cli": "^3.3.10",
-        "webpack-dev-server": "^3.10.1",
+        "webpack": "^5.92.0",
```

Comment:
can we check the bundle sizes between webpack 4 and webpack 5?

Reply:
updated description with images